src/ResourceManager/Sql/Commands.Sql/help/New-AzureRmSqlDatabase.md

```diff
@@ -285,24 +301,8 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -SampleName
-The name of the sample schema to apply when creating this database.
-
-```yaml
-Type: String
-Parameter Sets: (All)
-Aliases: 
-
-Required: False
-Position: Named
-Default value: None
-Accept pipeline input: False
-Accept wildcard characters: False
-```
-
 ### CommonParameters
-
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
```

Comment:
@akromm it does not appear to be `platyPS` intention to have the brackets around the link: https://github.com/PowerShell/platyPS/blob/b0c97c9de267d9ca46bfef17610a92648efa5069/src/Markdown.MAML/Resources/MarkdownStrings.resx#L175